packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -72,6 +83,27 @@ export const mapUserToApplicationContextUser = (user?: TFetchedUser) => {
     timeZone: user.timeZone || defaultTimeZone,
     projects: user.projects,
   };
+
+  // This property will only be populated when user has logged in using SSO
+  if (user.idTokenUserInfo) {
+    let additionalClaims: Record<string, unknown> = {};
+    try {
+      additionalClaims = JSON.parse(
+        user.idTokenUserInfo.additionalClaims || ''
+      );
+    } catch (error) {
+      console.warn(
+        '@commercetools-frontend/application-shell-connectors: Could not parse received user sso custom claims from server.',
```

Comment:
Should/could we report this to Sentry? Just thinking out loud.

Reply:
I guess we could use the [@commerctools-frontend/sentry](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/sentry) package to report the error.

My initial thought was to log it to the console so, if users experience this problem, they could see it and be aware of what's happening. Also, they could reach out to us and provide that error message.

Second thought: this error would only happen if we provide an invalid JSON from the MC backend, which I think is unlikely due to its implementation. But if it happens, it wouldn't be because of a user action but because of something wrong in our side.
Looking it that way, perhaps it may sense to report to Sentry.

@emmenko what do you think?

Reply:
Sure why not. Thanks for suggesting!

Reply:
Updated in this commit: 0b705034fac39ae971d8db39d1e646f6151910c9